src/ai/defaultai.cc

```diff
@@ -534,13 +534,21 @@ void DefaultAI::think() {
 					verb_log_info_time(gametime, "Conquered warehouses: %d / %" PRIuS "\n", conquered_wh,
 					                   enemy_warehouses.size());
 				}
-				management_data.review(
-				   gametime, player_number(), player_statistics.get_player_land(player_number()),
-				   player_statistics.get_enemies_max_land(),
-				   player_statistics.get_old60_player_land(player_number()), attackers_count_,
-				   soldier_trained_log.count(gametime),
-				   player_statistics.get_player_power(player_number()),
-				   count_productionsites_without_buildings(), first_iron_mine_built);
+
+				// how many types of mines have at least one finished mine? So can be up to 4 now
+				uint16_t finished_mines_type = 0;
+				for (auto const& mt : mines_per_type) {
```

Comment:
codecheck is complaining this should be (const auto& mt:... )

Reply:
should be fixed